lib/commentOnClosedIssue.js

```diff
@@ -85,4 +85,4 @@ commentOnClosedIssue._implementation = function (issueUrl, commentsUrl, reposito
         });
 };
 
-commentOnClosedIssue._googleLinkRegex = /https?:\/\/groups\.google\.com[^\s.,:)\\]*/ig;
+commentOnClosedIssue._googleLinkRegex = /https?:\/\/groups\.google\.com\/forum\/(\?\w+=\w+)?#!(topic|msg)\/cesium-dev\/(\w+\/?)+/ig;
```

Comment:
I think this is too strict now, for example these are all valid forum links that are not caught by this regex.

* https://groups.google.com/forum/?fromgroups#!topic/cesium-dev/bCzxAPBtX9k 
* https://groups.google.com/forum?fromgroups#!topic/cesium-dev/bCzxAPBtX9k 
* https://groups.google.com/d/msg/cesium-dev/bCzxAPBtX9k/EUGYilpgDgAJ



Perhaps we can simply check for `groups.google.com` `cesium-dev/` and make sure something is after `cesium-dev/`?